config/ember-try.js

```diff
@@ -69,6 +69,15 @@ module.exports = function () {
             },
           },
         },
+        {
+          name: 'ember-lts-3.28',
+          npm: {
+            devDependencies: {
+              'ember-data': '~3.28.0',
```

Comment:
I see that we add `ember-data` in all the other scenarios too, but do we actually need that in this addon?

Reply:
Yeah, good point – likely copy/paste artifact and could be removed completely.